templates/taxi/driver_list.html

```diff
@@ -5,10 +5,19 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+        {% if driver.id == user.id %}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Ensure that the URL pattern 'taxi:driver-detail' accepts a 'pk' parameter. If the URL pattern does not match, it will result in a NoReverseMatch error.